src/components/top-navigation/TopNav.jsx

```diff
@@ -1,21 +1,36 @@
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import { BiUserCircle } from "react-icons/bi";
+import { MdLogout } from "react-icons/md";
 import "./top-nav.css";
+import { useAuth } from "../../context/auth-context";
 
 const TopNav = () => {
+   const navigate = useNavigate();
+   const { authState, logoutUser } = useAuth();
+
    return (
-      <>
-         <nav className="main-nav">
-            <Link to="/">
-               <span className="h5 brand">hhTube</span>
-            </Link>
+      <nav className="main-nav">
+         <Link to="/">
+            <span className="h5 brand">hhTube</span>
+         </Link>
+         {!authState.isLoggedIn ? (
             <BiUserCircle
                color="white"
                size="2.15rem"
+               title="login"
+               style={{ cursor: "pointer" }}
```

Comment:
you can use class to give style to react icons also

Reply:
Thanks for the suggestion, will work on it.